src/pages/Subreddit/Subreddit.js

```diff
@@ -0,0 +1,48 @@
+import React, {useEffect, useState} from 'react';
+import {Link, useParams} from "react-router-dom";
+import axios from "axios";
+import './Subreddit.css'
+
+
+function Subreddit() {
+    const { id } = useParams();
+    const [childReddit, setChildReddit] = useState({})
```

Comment:
Ook dus hier de default waarde aanpassen
```suggestion
    const [childReddit, setChildReddit] = useState();
```